addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -385,3 +386,16 @@ registry.category("web_tour.tours").add("PosCategoriesOrder", {
             },
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("PosSaveOrderlinesIndexedDB", {
```

Comment:
what exactly gets tested here ?

Reply:
It will test if orderlines get saved in IndexedDB.

Reply:
if i add a product to the cart and refresh the page i still have the orderline. The info is kept. I agree that if i change the quantity and then refresh, the qty is lost. Maybe we can introduce this in the test

Reply:
This PR fixes the issue where, in some cases (in this test, when there is no tax on the product), the order line wasn't kept in IndexedDB. This issue is not related to the quantity. It saves the whole orderline in IndexedDB, and it's not possible to save an order line in IndexedDB, then load it, but have the quantity missing.

Reply:
the issue i am referring to is the following ( can be observed in your branch, but i think that it might start in an earlier version ) :

1. Open pos
2. Click on a product
3. Observe that it is added to the cart with qty1
4. Click on `2` on the numpad
5. observe that the qty is now 2
6. refresh the page
==> 7. observe that the quantity is now 1, instead of 2 <==

Reply:
Yes, I tested it. It's a different issue. The problem is that when you modify the quantity, it doesn't sync to IndexedDB. I think it's better to address this in a separate PR.

Reply:
ok, let's do that then